### ISSUE TITLE
Fails to load under Python 3

### ISSUE THREAD

**Issue by samhh** [nolsto/beets-follow#6]
As of 1.4.1, Beets has some Python 3 support. I reinstalled both it and this plugin in pip3 and I get an error running `beet` only with this plugin enabled, as follows:

```
** error loading plugin follow:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/beets/plugins.py", line 254, in load_plugins
    namespace = __import__(modname, None, None)
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/beetsplug/follow.py", line 40
    except ui.UserError, e:
                       ^
SyntaxError: invalid syntax
```

**Comment by apiontek** [comment]
I'm experiencing the same issue under python 3 with beets 1.4.3.

**Comment by nolsto** [comment]
I've commited 768ec10 which provides the plugin with support for both Python 2 & 3. I'll try to publish a new package at PyPI, but in the meantime you could try cloning the master branch into your beets `pluginpath` and try it out.

**Comment by nolsto** [comment]
Released v1.1 and published to PyPI.
